File: paradicms_etl/namespaces/time.py
from rdflib import URIRef
from rdflib.namespace import DefinedNamespace, Namespace


class TIME(DefinedNamespace):
    _fail = True
    _NS = Namespace("http://www.w3.org/2006/time#")

    # Properties
    day: URIRef
    hour: URIRef
    minute: URIRef
    month: URIRef
    year: URIRef

    # Resources
    DateTimeDescription: URIRef

File: lib/models/src/vocabularies/TIME.ts
import Namespace from "../Namespace";

export class TIME {
  static NS = Namespace("http://www.w3.org/2006/time#");

  // Properties
  static day = TIME.NS("day");
  static hour = TIME.NS("minute");
  static minute = TIME.NS("minute");
  static month = TIME.NS("month");
  static second = TIME.NS("second");
  static year = TIME.NS("year");

  // Resources
  static DateTimeDescription = TIME.NS("DateTimeDescription");
}
